### ISSUE TITLE
can't use metavars in golang imports

### ISSUE THREAD

**Issue by ievans** [semgrep/semgrep#638]
This idiom works in python and javascript, but not golang: https://semgrep.live/8pB

```
    pattern: import ( $X )
```
fails to parse

**Comment by aryx** [comment]
What do you want $X to bind to? If it's one of the module string, fine, but then you will not be able to reuse the same metavariable later.
Or you just want to check whether there's an import directive? In that case maybe import (...) would be better?

**Comment by nbrahms** [comment]
Fixed by #1305 and #1310. Just FYI, due to what constitutes valid Go code, you will need to match with:

```go
  import "$X"
```

but that will match every import in:

```go
  import (
    "fmt"
    "http"
    ...etc.
  )
```
